app/views/response/_response_analysis.html.erb

```diff
@@ -0,0 +1,415 @@
+<!DOCTYPE html>
```

Comment:
add coloring based on positive and negative (green and red)
add hover effects to explain what the metrics are.
add configurable text file to let professor set preferences for what metrics are being used